certified-connectors/TrueDialog SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1762 @@
+                  "default": true,
+                  "x-ms-visibility": "internal"
+                }
+              },
+              "required": [
+                "URL",
+                "CallbackType",
+                "Active"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/account/{accountId}/callback/-13": {
+      "x-ms-notification-content": {
+        "description": "Invalid Targets",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "CallbackTimestamp": {
+              "type": "string",
+              "description": "Date and time the callback has been sent. Format: yyyy-MM-ddTHH:mm:ss",
+              "title": "Callback Timestamp"
+            },
+            "CallbackToken": {
+              "type": "string",
+              "description": "CallbackToken",
+              "x-ms-visibility": "internal"
+            },
+            "CallbackType": {
+              "type": "integer",
+              "format": "int32",
+              "description": "CallbackType",
+              "title": "",
+              "x-ms-visibility": "internal"
+            },
+            "CallbackURL": {
+              "type": "string",
+              "description": "CallbackURL"
+            },
+            "TransactionId": {
+              "type": "string",
+              "description": "TransactionId"
+            },
+            "AccountId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "AccountId"
+            },
+            "ActionId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "ActionId"
+            },
+            "RunId": {
+              "type": "integer",
+              "format": "int32",
+              "description": "RunId"
+            },
+            "InvalidTargets": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "Target": {
+                    "type": "string",
+                    "description": "The target that has been filtered out",
+                    "title": "Target"
+                  },
+                  "ReasonId": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "ID of the reason. 0 - Invalid, 1 - Opted Out, 2 - Duplicate, 3 - Blocked, 4 - Undeliverable, 5 - Single Use",
+                    "title": "ReasonId"
+                  },
+                  "Reason": {
+                    "type": "string",
+                    "description": "Short description of a reason",
+                    "title": "Reason"
+                  }
+                }
+              },
+              "description": "InvalidTargets"
+            }
+          }
+        }
+      },
+      "put": {
+        "responses": {
+          "201": {
+            "description": "Invalid Targets callback created"
```

Comment:
@TrueDialogDynamics Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.